governance/roles.md

```diff
@@ -87,21 +89,8 @@ Leads, any Chair may act as Technical Lead.
   * Asking for new proposals to be made to address an identified need.
```

Comment:
This is vague and also feels that long-term planning as well ensuring the work stays current and relevant is a shared responsibility across all leadership roles